search/controllers/api/__init__.py

```diff
@@ -47,16 +51,16 @@ def search(params: MultiDict) -> Tuple[Dict[str, Any], int, Dict[str, Any]]:
 
     # parse advanced classic-style queries
```

Comment:
Confused about this -- isn't the controller for classic queries ``classic_query()`` (below)?

Reply:
When designing the NG API and Classic API, we discussed having feature parity between the two.

The `query` parameter in the NG API is meant to be a drop-in replacement for the multifield queries supported by the Classic API `search_query`. I added it for a migration pathway that would literally share the same Classic API Query formulation.

I haven't yet revised this to match the full implementation, so it does need attention, and I'll highlight the design considerations in revised comments.

Reply:
Gotcha. Would be good to screen off from this first PR, but good to highlight next steps

Reply:
Expanded the comment about the "migration pathway" to clarify why that code exists.

Reply:
I screened off the actual implementation with a `TODO` for a future PR.